homeassistant/components/glances/sensor.py

```diff
@@ -21,21 +22,21 @@
     TEMP_CELSIUS,
     Platform,
 )
-from homeassistant.core import CALLBACK_TYPE, HomeAssistant, callback
+from homeassistant.core import HomeAssistant, callback
 from homeassistant.helpers import entity_registry
-from homeassistant.helpers.dispatcher import async_dispatcher_connect
 from homeassistant.helpers.entity import DeviceInfo
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
 
-from . import GlancesData
-from .const import CPU_ICON, DATA_UPDATED, DOMAIN
+from . import GlancesDataUpdateCoordinator
+from .const import CPU_ICON, DOMAIN
 
 
 @dataclass
 class GlancesSensorEntityDescription(SensorEntityDescription):
     """Describe Glances sensor entity."""
 
-    type: str | None = None
+    type: str = ""
```

Comment:
`type` looks like it's required for required. Then we should use a mixin dataclass to define the attribute.

Example:
https://github.com/home-assistant/core/blob/1d9bbdcbf3d1d677f50d2c1b6a0f7568a2a20281/homeassistant/components/wled/sensor.py#L34-L47